WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1032,6 +1032,12 @@ public boolean isNewPost() {
         @Override
         protected Void doInBackground(Void... params) {
 
+            if (!mSite.getHasCapabilityPublishPosts()) {
+               if (PostStatus.fromPost(mPost) != PostStatus.DRAFT && PostStatus.fromPost(mPost) != PostStatus.PENDING) {
```

Comment:
We could save a `fromPost` call here by saving the result, but that's a pretty nitpicky change.